huxley/core/models.py

```diff
@@ -356,11 +356,11 @@ class Meta:
 
 class Delegate(models.Model):
     school        = models.ForeignKey(School, related_name='delegates', null=True)
-    assignment    = models.ForeignKey(Assignment, related_name='delegates', blank=True)
+    assignment    = models.ForeignKey(Assignment, related_name='delegates', blank=True, null=True)
     name          = models.CharField(max_length=64)
-    email         = models.EmailField(blank=True)
+    email         = models.EmailField(blank=True, null=True)
     created_at    = models.DateTimeField(auto_now_add=True)
-    summary       = models.TextField(default='', null=True)
+    summary       = models.TextField(default='', blank=True, null=True)
```

Comment:
We should remove `null=True`. Unless we have some preference for this being null instead of an empty string when nothing is submitted.

https://docs.djangoproject.com/en/1.8/ref/models/fields/#null